src/componentes/CourseSubjects/index.jsx

```diff
@@ -0,0 +1,53 @@
+import Button from "../Button";
+import CourseCard from "./CourseCard";
+
+function CourseSubjects({ courseData }) {
+  return (
+    <div className="pt-10">
+      <h1 className="text-xl">{courseData.nome}</h1>
+      {/* <div>
+        <Button>Obrigatórias</Button>
+        <Button>Optativas</Button>
+      </div> */}
+      <div className="bg-primary-50 p-5 z-10 m-5 shadow-lg rounded-lg">
```

Comment:
```suggestion
      <div className="bg-primary-50 p-5 z-10 m-5 shadow-lg rounded-lg overflow-auto">
```

Para deixar o scroll apenas para a div e não para a página toda

![image](https://github.com/ic045-sistemaacademico-2023/frontend-sistema-academico-ic-ufba/assets/55323701/b5b6be38-e614-4ca2-b102-4f8a590bfed5)